importer/docker_local_import.py

```diff
@@ -330,7 +330,12 @@ def import_html(paths):
     ctx = sycamore_init()
     (
         ctx.read.binary(paths, binary_format="html", filter_paths_by_extension=False)
-        .partition(partitioner=HtmlPartitioner())
+        .partition(
+            partitioner=HtmlPartitioner(
+                extract_tables=True,
+                text_chunker=TextOverlapChunker(chunk_token_count=4000, chunk_overlap_token_count=400),
```

Comment:
There is an alternative to this.  When you feed search results to an LLM, you always include the elements surrounding the results.  That might give applications more control.